server/webapp/WEB-INF/rails/webpack/views/pages/new_agents/environment_dropdown_button.tsx

```diff
@@ -15,16 +15,21 @@
  */
 
 import {ApiResult} from "helpers/api_request_builder";
+import _ from "lodash";
 import m from "mithril";
 import Stream from "mithril/stream";
 import {Agent} from "models/new_agent/agents";
 import {DropdownAttrs, Primary} from "views/components/buttons";
 import {TriStateCheckboxField} from "views/components/forms/input_fields";
 import Style from "views/pages/new_agents/index.scss";
+import {StaticAgentsVM} from "../../../models/new_agent/agents_vm";
```

Comment:
`../../../models/new_agent/agents_vm` can be changed to `models/new_agent/agents_vm`